mockup/patterns/structure/js/views/addmenu.js

```diff
@@ -14,7 +14,7 @@ define([
     initialize: function(options) {
       var self = this;
       ButtonGroup.prototype.initialize.apply(self, [options]);
-      self.app.on('context-info-loaded', function(data) {
+      $('body').on('context-info-loaded', function(data) {
```

Comment:
the event is now triggered on the body, so that I can listen outside of the pattern for it.
(the event didn't bubble up, don't know why..)